frontend/src/pages/Dashboard.jsx

```diff
@@ -66,7 +66,7 @@ export default function Dashboard() {
           <div>Zetsy</div>
           <div className="flex flex-row gap-2">
             <button
-              onClick={() => window.open("https://zetsy.store", "_blank")}
+              onClick={() => window.open(`https://${selected.storeUrl}`, "_blank")}
```

Comment:
Dynamic Domain Routing must be done.

Reply:
I do not know more about dynamic routing and sub-domains, other than that it seems pretty good.